starknet_py/net/models/deployer_addresses.py

```diff
@@ -0,0 +1,23 @@
+from typing import Optional
+
+from starknet_py.net.networks import Network, MAINNET, TESTNET
+
+
+def deployer_address_from_network(
+    net: Network, deployer_address: Optional[int] = None
+) -> int:
+    # FIXME: change to real addresses once deployed
+    mapping = {
+        MAINNET: 123,
+        TESTNET: 456,
+    }
```

Comment:
Change when deployers will be deployed on mainnet and testnet

Reply:
I don't think that this function is useful anymore. There is an issue in devnet to change the pedeployed deployer address to `0x041a78e741e5aF2fEc34B695679bC6891742439f7AFB8484Ecd7766661aD02BF` to match testnet
https://github.com/Shard-Labs/starknet-devnet/issues/326

Reply:
Hmm, but you still need to resolve the address for mainnet/testnet right?

Reply:
nope, there is a link in the devnet issue, please check it

Reply:
Let's remove it then